mlir/test/Target/LLVMIR/erase-dangling-constants.mlir

```diff
@@ -1,3 +1,4 @@
+// REQUIRES: asserts
```

Comment:
I don't believe we test on --debug output. I would rather remove this test.

Reply:
Removing with PR https://github.com/llvm/llvm-project/pull/83189